client/src/containers/SignUp/SignUpForm/SignUpForm.tsx

```diff
@@ -0,0 +1,132 @@
+import React from "react";
+import styled from "styled-components";
+import * as Yup from "yup";
+import { Formik, Form, Field, FormikErrors } from "formik";
+import TextHelper from "../../../components/TextHelper/TextHelper";
+import TextField from "../../../components/TextField/TextField";
+import Button from "../../../components/Button/Button";
+
+const StyledFormikForm = styled(Form)`
+  display: grid;
+  grid-gap: 15px;
+  width: 100%;
+  max-width: 400px;
+  padding: 20px 0;
+`;
+
+const SignupSchema = Yup.object().shape({
+  firstName: Yup.string()
+    .min(2, "First name length must be at least 2 characters")
+    .required("First name is required"),
+  lastName: Yup.string()
+    .min(2, "First name length must be at least 2 characters")
+    .required("First name is required"),
+  email: Yup.string()
+    .email("Invalid email")
+    .required("Email is required"),
+  password: Yup.string()
+    .min(6, "Password length must be at least 6 characters")
+    .required("Password is required"),
+  passwordRepeat: Yup.string()
+    .oneOf([Yup.ref("password"), null], "Passwords must match")
+    .min(6, "Password repeat length must be at least 6 characters")
+    .required("Password repeat is required")
+});
+
+interface ClientErrors {
+  errors: FormikErrors<Values>;
+}
+
+const FormErrors: React.FC<ClientErrors> = ({ errors }) => {
+  if (Object.keys(errors).length > 0) {
+    return (
+      <TextHelper error component="div">
+        {Object.values(errors).map((error, index) => (
+          <p key={index} style={{ marginBottom: "5px" }}>
+            - {error}
+          </p>
+        ))}
+      </TextHelper>
+    );
+  } else {
+    return (<></>)
+  }
+};
+
+interface Values {
```

Comment:
It looks the same like in file `SignUp.tsx `

Can we export it to external file and import in both files?

Reply:
Done